src/org/openjump/core/rasterimage/AddRasterImageLayerWizard.java

```diff
@@ -215,6 +216,25 @@ private void addImage(WorkbenchContext context, Envelope envelope,
         } catch (Exception e) {
             Logger.error(e);
         }
+        // [Giuseppe Aruta 2024_08_19]
+        // This part of code allows to read and apply a style to a RasterImageLayer.
+        // The style must be stored as SLD file with the same name of the layer.
+        if (rLayer.getNumBands() == 1) {// Currently OpenJUMP can read/write symbology only for
+            // monoband raster files
+            String sldS = org.openjump.util.UriUtil.removeExtension(imageFileName) + ".sld";
+            // String sldS = new File(imageFileName).getAbsolutePath().replace("tif",
+            // "sld");

```

Comment:
please remove the commented code. it's not helpful.

apart from that. no objections. nice indention :)